daffodil-core/src/main/scala/org/apache/daffodil/core/dpath/Expression.scala

```diff
@@ -2546,18 +2544,17 @@ case class FNZeroArgExpr(
   nameAsParsed: String,
   fnQName: RefQName,
   resultType: NodeInfo.Kind,
-  argType: NodeInfo.Kind,
-  constructor: (CompiledDPath, NodeInfo.Kind) => RecipeOp
+  constructor: () => RecipeOp
 ) extends FunctionCallBase(nameAsParsed, fnQName, Nil) {
 
   override lazy val inherentType = resultType
 
-  override def targetTypeForSubexpression(childExpr: Expression): NodeInfo.Kind = argType
+  override def targetTypeForSubexpression(childExpr: Expression): NodeInfo.Kind = NodeInfo.Exists
```

Comment:
This function should never be called since `FNZeroArgExpr`'s never have sub expressions. So instead of returning `NodeInfo.Exists`, we probably should just call `Assert.impossible()` wrapped with `// $COVERAGE-OFF$` and `// $COVERAGE-ON$` comments to to disable code coverage.

Reply:
Done!